Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "trunk-analytics-cli"
 edition = "2021"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
Increase minor version since we are adding new functionality in a backwards compatible way.

Reply:
Can cargo.toml be generated from git tag?

Reply:
That's a great question, I'm not sure.